app/controllers/assignments_controller.rb

```diff
@@ -7,6 +7,8 @@ class AssignmentsController < ApplicationController
   responders :flash
   before_action { authorize! }
 
+  authorize :test_id, through: :test_id_param
```

Comment:
Name this `test_run_id` and `test_run_id_param` to make it consistent with the model name